health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/presentation/exercise/ExerciseScreen.kt

```diff
@@ -81,6 +103,54 @@ fun ExerciseRoute(
     }
 }
 
+/**
+ * Shows an error that occured when starting an exercise
+ */
+@Composable
+fun ErrorStartingExerciseScreen(
+    onRestart: () -> Unit,
+    uiState: ExerciseScreenState
+) {
+    Column(
```

Comment:
Feels like we could use a precanned component like Alert?

Reply:
Done. Replaced with Horologist AlertDialog and updated screenshot in PR description.